tests/unit/states/test_mount.py

```diff
@@ -62,12 +62,11 @@ def test_mounted(self):
         mock_str = MagicMock(return_value='salt')
         mock_user = MagicMock(return_value={'uid': 510})
         mock_group = MagicMock(return_value={'gid': 100})
-        umount1 = ("Forced unmount because devices don't match. "
-                   "Wanted: /dev/sdb6, current: /dev/sdb5, /dev/sdb5")
         with patch.dict(mount.__grains__, {'os': 'Darwin'}):
             with patch.dict(mount.__salt__, {'mount.active': mock_mnt,
                                              'cmd.run_all': mock_ret,
-                                             'mount.umount': mock_f}):
+                                             'mount.umount': mock_f}), \
+                    patch('os.path.exists', MagicMock(return_value=True)):
```

Comment:
Why is this being patched? It seems like this changes the behavior of this test. Could you please clarify?